packages/core/content-releases/admin/src/pages/tests/ReleaseDetailsPage.test.tsx

```diff
@@ -53,11 +60,53 @@ describe('Release details page', () => {
 
     const deleteButton = screen.getByRole('button', { name: 'Delete' });
     expect(deleteButton).toBeInTheDocument();
+
+    const createdByAuthor = screen.getByText(/by Admin Admin/i);
+    expect(createdByAuthor).toBeInTheDocument();
+
+    const paginationCombobox = screen.queryByRole('combobox', { name: /entries per page/i });
+    expect(paginationCombobox).not.toBeInTheDocument();
   });
 
-  it('shows empty content if there are no entries', async () => {
+  it('renders the details page with actions', async () => {
+    server.use(
+      rest.get('/content-releases/:releaseId', (req, res, ctx) =>
+        res(ctx.json(mockReleaseDetailsPageData.withActionsHeaderData))
+      )
+    );
+
+    server.use(
+      rest.get('/content-releases/:releaseId/actions', (req, res, ctx) =>
+        res(ctx.json(mockReleaseDetailsPageData.withActionsBodyData))
+      )
+    );
```

Comment:
Do you think it's beneficial for growth if these default success responses were in the server by default?